src/mlpack/core/tree/rectangle_tree/discrete_hilbert_value.hpp

```diff
@@ -182,6 +182,14 @@ class DiscreteHilbertValue
    */
   DiscreteHilbertValue& operator=(const DiscreteHilbertValue& val);
 
+   /**
+   * Move the local Hilbert object.
+   *
+   * @param val The DiscreteHilbertValue object from which the dataset
+   *    will be copied.
+   */
+  DiscreteHilbertValue& operator=(DiscreteHilbertValue&& val);
```

Comment:
I would use `other` instead of `val` here as well, it's what you used in the implementation as well.

Reply:
sure